docs/connectors/mongodb.asciidoc

```diff
@@ -240,94 +240,111 @@ Let's look at what a _create_/_read_ event value might look like for our `custom
 {
     "schema": {
       "type": "struct",
-      "optional": false,
-      "name": "fulfillment.inventory.customers.Envelope",
-      "version": 1,
       "fields": [
         {
-          "field": "op",
-          "type": "string",
-          "optional": false
-        },
-        {
-          "field": "after",
           "type": "string",
-          "optional": true
+          "optional": true,
+          "name": "io.debezium.data.Json",
+          "version": 1,
```

Comment:
Any idea where this is coming from now, @jpechane? It wasn't there before.

Reply:
https://issues.jboss.org/browse/DBZ-100

Reply:
Sorry, I meant the "version" field, it wasn't there before. But then there was one for the Envelope type. This might have changed well before DBZ-1235 and never was updated in the docs. But I'm curious what's going on. The issue you linked seems not related?

Reply:
@gunnarmorling Looking for the vwriosn, I thought the type name was the problem so that's why I pointed to the origina l issue.

Reply:
@gunnarmorling Check Json datatype, it explicitly sets version to 1.

Reply:
Ok, I see. Probably doesn't matter then. On the envelope version, this was consciously removed in DBZ-466 a while ago, we just never updated these examples here. All good.